Darwin/StitchCore/StitchCore/Core/Auth/StitchUser.swift

```diff
@@ -42,11 +42,10 @@ public protocol StitchUser: CoreStitchUser {
 
     // Disabled line length rule due to https://github.com/realm/jazzy/issues/896
     // swiftlint:disable line_length
-
     /**
-     * Links this `StitchUser` with a new identity, where the identity is defined by the credential specified as a
-     * parameter. This will only be successful if this `StitchUser` is the currently authenticated `StitchUser` for the
-     * client from which it was created.
+     * Links the currently authenticated `StitchUser` with a new identity, where the identity is defined by the credential
+     * specified as a parameter. This will only be successful if this `StitchUser` is the currently authenticated
```

Comment:
[minor] you should add the `isLoggedIn` and `lastAuthActivity` properties to this protocol, otherwise they're not exposed to users